### PR TITLE
ref: add cloud versioning examples

### PR DESCRIPTION
**Pull request by dberenbaum**
@jorgeorpinel WDYT about these in the ref vs the guide?

### CODE REVIEW

File: content/docs/command-reference/import-url.md [hunk 1]
```diff
@@ -439,3 +440,49 @@ Use `dvc update --to-remote` to bring the import up to date in remote storage,
...
+DVC knows that your cloud storage is already versioning these files, so it won't
+push them to the DVC remote.
+
+```dvc
+$ dvc push
+Everything is up to date.
+```
+
+During `dvc pull`, these files will be pulled from their original source
+location rather than the DVC remote.
```
Comment (dberenbaum) [inline content/docs/command-reference/import-url.md:488]: This is broken right now: https://github.com/iterative/dvc/issues/8932

**Review by jorgeorpinel — APPROVED** [review]
I'd just also link from https://dvc.org/doc/user-guide/data-management/cloud-versioning.

File: content/docs/command-reference/push.md [hunk 2]
```diff
@@ -274,3 +274,90 @@ Cache and remote 'r1' are in sync.
...
+          r1:
+            etag: b656f1a8273d0c541340cb129fd5d5a9
+            version_id: T6rFr7NSHkL3v9tGStO7GTwsVaIFl42T
+      - size: 6728772
+        md5: 9ca281786366acca17632c27c5c5cc75
+        relpath: train.tsv
+        cloud:
+          r1:
+            etag: 9ca281786366acca17632c27c5c5cc75
+            version_id: XaYsHQHWK219n5MoCRe.Rr7LeNbbder_
```
Comment (jorgeorpinel) [inline content/docs/command-reference/push.md:362]:
```suggestion
    ...
    outs:
    - path: data/prepared
      files:
      - size: 1708591
        md5: b656f1a8273d0c541340cb129fd5d5a9
        relpath: test.tsv
        cloud:
          versioned_store:
            etag: b656f1a8273d0c541340cb129fd5d5a9
            version_id: T6rFr7NSHkL3v9tGStO7GTwsVaIFl42T
      - size: 6728772
        md5: 9ca281786366acca17632c27c5c5cc75
        relpath: train.tsv
        cloud:
          versioned_store:
            etag: 9ca281786366acca17632c27c5c5cc75
            version_id: XaYsHQHWK219n5MoCRe.Rr7LeNbbder_
```

Not sure we need all this in the example though, seems too deep. Maybe just mention `version_id` and/or consider expanding the dvc.lock sample in https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#dvclock-file instead.

Comment (jorgeorpinel) [reply content/docs/command-reference/push.md:362]: p.s. yeah isn't that file missing `version_id`, actually?

**Review by jorgeorpinel — COMMENTED** [review]
Just some ideas

File: content/docs/command-reference/import-url.md (hunk 1 above)
Comment (dberenbaum) [reply content/docs/command-reference/import-url.md:488]: Fixed

**Pull request merged**